thirdparty/libedit-3.1/examples/fileman

```diff
@@ -1,4 +1,4 @@
-#! /nix/store/mpgbs7f5x9k5cj5pa6hqk5rf7ph5hx6d-bash-4.4-p23/bin/bash
```

Comment:
`./configure` should already replace these paths with whatever makes sense for the particular system I think. If not, they should be generated by some other command in autotools

Also why is this file used in the first place?

Reply:
No changes have been committed to thirdparty any more (except for protobuf-2.6.1 deletion), so this should be fixed.